don't use the custom backend, we do the ui compilation in pre_build

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -69,11 +69,7 @@
   "setuptools>=64",
   "setuptools-scm>=8"
 ]
-# build-backend = 'setuptools.build_meta'
-# in-tree build backend, wrapper around setuptools, just to run ui-compile task
-# See also https://setuptools.pypa.io/en/latest/build_meta.html#dynamic-build-dependencies-and-other-build-meta-tweaks
-build-backend = "setuptools_wrapper"
-backend-path = ["src/tools"]
+build-backend = 'setuptools.build_meta'
 
 [tool.setuptools.exclude-package-data]
 "*" = ['*.png', '*.ui', '*.qrc' ]
